ROMFS/px4fmu_common/init.d-posix/px4-rc.mavlink

```diff
@@ -32,7 +32,7 @@ mavlink start -x -u $udp_onboard_payload_port_local -r 4000 -f -m onboard -o $ud
 mavlink start -x -u $udp_onboard_gimbal_port_local -r 400000 -m gimbal -o $udp_onboard_gimbal_port_remote
 
 # To display for SIH sitl
-if [ "$PX4_SIMULATOR" = "sihsim" ]; then
+if [ "$PX4_SIMULATOR" == "sihsim" ]; then
```

Comment:
Actually this is correct for bash within [ ], you use == when it's [[  ]]. Yes this is confusing.

Reply:
well on MacOS it was not getting inside the if statement. Now, with the double == it works. Can we keep as it is?

Reply:
No preference as long as it works everywhere. You can check some of these things with shellcheck (runs in CI). https://www.shellcheck.net/

Reply:
Which airframe are you trying to launch?

Reply:
I tried with quadx.
It seems the airplane doesn't work on v1.14. It seems there are problems with the actuator mapping.